daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1160,11 +1201,11 @@ void IEEE1588Port::processEvent(Event e)
 		putLastPDelayLock();
 		break;
 	case PDELAY_RESP_RECEIPT_TIMEOUT_EXPIRES:
-		if (!automotive_profile) {
+		if (!clock->getForceAsCapable()) {
```

Comment:
Personal preference: I think it would be safer to check if the automotive profile is enabled before or along with all of your conditionals for automotive feature variables. That way, someone doesn't inadvertently change the behavior of the standard profile by changing the defaults of the automotive profile. E.g. What will happen in the standard profile here if someone changes the default of the automotive forceAsCapable variable to true?

Reply:
All automotive profile configurations are private members of IEEE1588Clock class and we do not provide setter interface for these properties. So for now, it is "safe" outside the IEEE1588Clock class. But I think you're right, future code changes may break our assumption. So I "double-check" the automotiveProfile configuration in every getter function of the automotive profile configurations.